z3c/dependencychecker/package.py

```diff
@@ -121,6 +121,8 @@ def get_required_dependencies(self):
         this_package = self._get_ourselves_from_working_set()
         requirements = this_package.requires()
         for requirement in requirements:
+            if requirement.project_name == "Zope":
+                continue
```

Comment:
Would a `logger.debug("Ignoring 'Zope' requirement")` or so be a good idea?

Reply:
Should we do the same on the extras requirements? As we do it within a list comprehension it is a bit trickier to do it though :-/

Reply:
Ouch, yes, that's trickier.
Well, I guess Zope will normally be a normal requirement, not somewhere in the extras? Fine enough if the main usecase is being logged.

Reply:
Yes, that's what I though as well, but wanted to double check with you 😄